dali/test/python/test_operator_coord_flip.py

```diff
@@ -1,98 +0,0 @@
-# Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Again, this change is related to the copying of the new file over the older version ported before.